reglemente.tex

```diff
@@ -476,7 +495,7 @@ \section{Aktivitetsutskottet}
 	\begin{itemize}
 		\item ShopaholiC
```

Comment:
```suggestion
		\item Shopaholic
```

Reply:
Har comittat, bör squahas in men vill ej rebasea nu